site2/docs/io-elasticsearch-sink.md

```diff
@@ -16,11 +48,45 @@ The configuration of the ElasticSearch sink connector has the following properti
 |------|----------|----------|---------|-------------|
 | `elasticSearchUrl` | String| true |" " (empty string)| The URL of elastic search cluster to which the connector connects. |
 | `indexName` | String| true |" " (empty string)| The index name to which the connector writes messages. |
+| `schemaEnable` | Boolean | false | false | Turn on the Schema Aware mode. |
+| `createIndexIfNeeded` | Boolean | false | false | Manage index if missing. |
+| `maxRetries` | Integer | false | 1 | The maximum number of retries for elasticsearch requests. Use -1 to disable it.  |
```

Comment:
ElasticSearch
Please check all

Reply:
If you look at the Elasticsearch repo, the code, the doc always uses Elasticsearch, not ElasticSearch.

Reply:
Oh yes, sorry my bad, it should be `Elasticsearch`

Reply:
fixed all occurrences